pkg/lib/cockpit-components-password.jsx

```diff
@@ -98,7 +98,7 @@ export const PasswordFormFields = ({
                        labelIcon={password_label_info &&
                            <Popover bodyContent={password_label_info}>
                                <button onClick={e => e.preventDefault()}
-                                       className="pf-c-form__group-label-help">
+                                       className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-134728-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)